docs/profile_dependencies.md

```diff
@@ -0,0 +1,157 @@
+
+## Defining the Dependencies
+
+Before a profile can use controls from another profile, the to-be-included profile needs to be specified in the including profile’s `inspec.yml` file in the `depends` section. For each profile to be included, a location for the profile from where to be fetched and a name for the profile should be included. For example:
+
+    depends:
+    - name: linux-baseline
+      url: https://github.com/dev-sec/linux-baseline/archive/master.tar.gz
+    - name: ssh-baseline
+      url: https://github.com/dev-sec/ssh-baseline/archive/master.tar.gz
+
+InSpec supports a number of dependency sources.
+
+### path
+
+The `path` setting defines a profile that is located on disk. This setting is typically used during development of profiles and when debugging profiles.
+
+    depends:
+    - name: my-profile
+      path: /absolute/path
+    - name: another
+      path: ../relative/path
+
+### url
+
+The `url` setting specifies a profile that is located at an HTTP- or HTTPS-based URL. The profile must be accessible via a HTTP GET operation and must be a valid profile archive (zip, tar, or tar.gz format).
+
+    depends:
+    - name: my-profile
+      url: https://my.domain/path/to/profile.tgz
+    - name: profile-via-git
+      url: https://github.com/myusername/myprofile-repo/archive/master.tar.gz
+
+### git
+
+A `git` setting specifies a profile that is located in a git repository, with optional settings for branch, tag, commit, and version. The source location is translated into a URL upon resolution. This type of dependency supports version constraints via semantic versioning as git tags.
+
+For example:
+
+    depends:
+    - name: git-profile
+      git: http://url/to/repo
+      branch:  desired_branch
+      tag:     desired_version
+      commit:  pinned_commit
+      version: semver_via_tags
+
+### supermarket
+
+A `supermarket` setting specifies a profile that is located in a cookbook hosted on Chef Supermarket. The source location is translated into a URL upon resolution.
+
+For example:
+
+    depends:
+    - name: supermarket-profile
+      supermarket: supermarket-username/supermarket-profile
+
+Available Supermarket profiles can be listed with `inspec supermarket profiles`.
+
+### compliance
+
+A `compliance` setting specifies a profile that is located on the Chef Automate or Chef Compliance server.
+
+For example:
+
+    depends:
+      - name: linux
+        compliance: base/linux
+
+## Vendoring Dependencies
+
+When you execute a local profile, the `inspec.yml` file will be read in order to source any profile dependencies. It will then cache the dependencies locally and generate an `inspec.lock` file.
+
+If you add or update dependencies in `inspec.yml`, dependencies may be re-vendored and the lockfile updated with `inspec vendor --overwrite`
+
```

Comment:
On reflection,`Using Controls` may be a better break; so keep Dependencies on the profiles page and make a page on control inheritance.